docs/_static/mxnet-theme/navbar.html

```diff
@@ -10,8 +10,9 @@ <h1 id="logo-wrap">
         <span id="dropdown-menu-position-anchor-community">
           <a href="#" class="main-nav-link dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="true">Community <span class="caret"></span></a>
           <ul id="package-dropdown-menu-community" class="dropdown-menu navbar-menu">
-            <li><a class="main-nav-link" href="{{url_root}}community/index.html">Community</a></li>
-            <li><a class="main-nav-link" href="{{url_root}}community/collaborator.html">Collaborator</a></li>
+            <li><a class="main-nav-link" href="{{url_root}}community/index.html">Mailing List & Resources</a></li>
```

Comment:
Would this be too long?

Reply:
Do you mean "Mailing List & Resources" is too long? It seems ok on https://spark.apache.org/

Reply:
I agree, I think we should come up with something easier and better. Let me talk to Seb and get back.

Reply:
Let's keep it "Community" then. So the title of the menu list will be Community, and the three items in the dropdown will be Community, Contribute, Powered By. Does that work?

Reply:
@thinksanky Can we change "Mailing List & Resources" to "Community" and merge this?